__test__/Unit.test.js

```diff
@@ -1,34 +1,173 @@
-// Unit.test.js
-/*
-import {
-    sum
-} from '../code-to-unit-test/unitTest';
-*/
-const sum = require("../code-to-unit-test/unitTest");
-//Testing to see if the file works
-/*global test*/
-/*eslint no-undef: "error"*/
-test("Sum of 1 + 2", () => {
-  expect(sum(1, 2)).toBe(3);
+// Need to run the following commands to run the tests:
```

Comment:
So long as it's setup properly devs should be able to install all dependencies with npm install `npm install --save-dev jest-environment-jsdom` is unnecessary.

Reply:
Removed that instruction.